src/fiber/fiber.mli

```diff
@@ -387,3 +387,20 @@ type fill = Fill : 'a Ivar.t * 'a -> fill
     the scheduler, it should block waiting for an event and return an ivar to
     fill. *)
 val run : 'a t -> iter:(unit -> fill) -> 'a
+
+(** [Temp.Monad] instantiated to the Fiber monad. *)
```

Comment:
So far we've kept the `Fiber` module completely free of anything IO related. I'm a little sad to depart from this. Could we put this in a separate module/library?

Reply:
Yes, sure. Does `fiber_util` sound good?

Reply:
Sounds good to me

Reply:
Done.